### PR TITLE
Vulcan: Make Causes list visible above the fold

### PR DESCRIPTION
**Pull request by andyg0808**
This does what it says on the tin.

## QA Notes
Please confirm that it handles a page with no introduction text correctly (it should hide the link to the introduction), and that the icon is clear to you. Detailed spec: https://docs.google.com/document/d/1YhlRLq1C6sRU_nYC-Kf9eQ6CDGvjHZU_5zwo3NIqLuI/edit#bookmark=id.vyolc43ijwj

Closes https://github.com/iFixit/ifixit/issues/49392

### CODE REVIEW

**Comment by deltuh-vee** [comment]
dev_block ✊ 
The causes have been moved up 
CleanShot 2023-08-19 at 13 21 30
...but not the header.
CleanShot 2023-08-19 at 13 21 40

**Comment by ianrohde** [comment]
I'm working on getting clear definitions of design for this. I'll CR once those are clear.

**Comment by sctice-ifixit** [comment]
Mockup is at https://www.figma.com/file/QVLUEpDVoyiMM4MXEGq1L9/iFixit---Problems?type=design&node-id=201-65621&mode=design&t=mNCCcgDmO3qKEROU-4. The Causes header should be at the top, and there's a separator between the causes list and the intro / first cause.

image

**Comment by andyg0808** [comment]
un_dev_block :+1: 
This should follow the spec now.

**Comment by andyg0808** [comment]
Looks like @jarstelfox's TOC work has been merged; dev_block :+1: on integrating that.

**Comment by ianrohde** [comment]
This is a great test-case for this pull: `/Troubleshooting/Whirlpool_Washing_Machine/F3E1+Code/504703`

File: frontend/templates/troubleshooting/index.tsx [hunk 1]
```diff
@@ -35,6 +35,7 @@ import {
    HStack,
    SimpleGrid,
    useToken,
+   Heading,
```
Comment (ianrohde) [inline frontend/templates/troubleshooting/index.tsx:38]:
```suggestion
```

**Comment by jordycosta** [comment]
I don't seem to be seeing a causes list

https://react-commerce-git-vulcan-make-causes-list-visibl-c41228-ifixit.vercel.app/Troubleshooting/iPhone/Battery+Draining+Fast/480045

https://github.com/iFixit/react-commerce/assets/95656772/2aea80b0-5e27-4d96-bd39-d2f9bfdc15a2

**Comment by jarstelfox** [comment]
@jordycosta the causes list should be hidden on desktop, but visible on mobile

**Comment by andyg0808** [comment]
un_dev_block :+1: 
The mobile display should be fixed now.

**Comment by jordycosta** [comment]
QA 🟢 

[Page with no introduction](https://react-commerce-git-vulcan-make-causes-list-visibl-c41228-ifixit.vercel.app/Troubleshooting/iPhone/Won't+Turn+On/405189)
[Page with introduction](https://react-commerce-git-vulcan-make-causes-list-visibl-c41228-ifixit.vercel.app/Troubleshooting/iPhone/Battery+Draining+Fast/480045)

---

The troubleshooting page with an introduction looks good. The causes list is the first thing shown on mobile view and it has the introduction icon + section heading included in it. 

---

When a troubleshooting page has no introduction:

There is no longer an empty section (shown below) ✔️ 

firefox_UkMzoV3dKT

The introduction section is not in the _Causes list_ ✔️ 

firefox_yvhm2L5HYf

**Comment by jordycosta** [comment]
deploy_block 🟨 

I did want to point out that there seems to be more vertical spacing in between the **Introduction** and title/author section in comparison to the [figma mockup ](https://www.figma.com/file/jRG4EyQXRDMGfyLqut08KP/iFixit---Working-files?type=design&node-id=13996-318155&mode=design)

**Review by ianrohde — APPROVED** [review]
CR ⚪

**Review by ianrohde — DISMISSED** [review]
CR ⚪

**Comment by andyg0808** [comment]
un_deploy_block :+1:

**Comment by jordycosta** [comment]
QA 🟢 

Looks good!

No functionality lost from https://github.com/iFixit/react-commerce/pull/1925#issuecomment-1699852748 👍 

firefox_7xIk8qVLHX

**Review by ianrohde — APPROVED** [review]
CR ⚪

**Pull request merged**